pallets/vault-registry/src/mock.rs

```diff
@@ -360,6 +369,14 @@ pub const RICH_ID: VaultId<AccountId, CurrencyId> = VaultId {
 		wrapped: DEFAULT_WRAPPED_CURRENCY,
 	},
 };
+pub const ID_COLLATERAL_21: VaultId<AccountId, CurrencyId> = VaultId {
+	account_id: 6,
+	currencies: VaultCurrencyPair { collateral: XCM(1), wrapped: DEFAULT_WRAPPED_CURRENCY },
+};
+pub const ID_COLLATERAL_22: VaultId<AccountId, CurrencyId> = VaultId {
```

Comment:
Let's try to find better names. How about `COLLATERAL_2_VAULT_ID_1` and `COLLATERAL_2_VAULT_ID_2`? Also not perfect but maybe clearer

Reply:
We could then also rename `DEFAULT_ID` and `OTHER_ID` to `COLLATERAL_1_VAULT_ID_1` and `COLLATERAL_1_VAULT_ID_2`

Reply:
I agree, looking at it after a few days this is not clear anymore. Is better long but explicit.